requirements/requirements.txt

```diff
@@ -10,6 +10,6 @@ pydantic
 pyzmq
 safetensors
 torch
-transformers
+transformers>=4.41.2
```

Comment:
`ConversationalPipeline` and `Conversation` has been deprecated for a while and was removed in transformers 4.41.2. Do we really have to pin this version as minimum? These changes should be backward-compatible.

Reply:
Good point, unpinning.